ConsentViewController/Classes/GDPRConsentViewController.swift

```diff
@@ -221,11 +221,13 @@ typealias Meta = String
                 clearAllData()
                 localStorage.authId = authId
             }
+            localStorage.authId = authId
             sourcePoint.getMessage(native: native, consentUUID: gdprUUID, euconsent: euconsent, authId: authId, meta: localStorage.meta) { [weak self] messageResponse, error in
                 if let messageResponse = messageResponse {
                     self?.localStorage.consentUUID = messageResponse.uuid
                     self?.localStorage.meta = messageResponse.meta
                     self?.localStorage.userConsents = messageResponse.userConsent
+                    self?.localStorage.authId = authId
```

Comment:
I believe we don't need to add the authId to the localStorage again here.

Reply:
Forgot to remove, will remove with new commint